docs/source/notebooks/model_comparison.ipynb

```diff
@@ -1,25 +1,34 @@
 {
```

Comment:
<ul><li><s>THis</s> <strong>This</strong> approximation is base<strong>d</strong> on importance sampling</li><li>By default ArviZ<strong>,</strong> use<strong>s</strong> LOO, but WAIC is also available.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc3/pull/4042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>